app/src/main/java/com/teamagam/gimelgimel/app/message/view/ImageFullscreenActivity.java

```diff
@@ -83,10 +81,19 @@ private void hideControls() {
 
     private void showControls() {
         // Show the system bar
-        mDraweeView.setSystemUiVisibility(View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
+        mImageView.setSystemUiVisibility(View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
                 | View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION);
 
         mIsControlsVisible = true;
     }
 
+
+    private void loadImage(Uri imageUri) {
```

Comment:
This is the same code piece as in setImageUrl at the MEssageViewHolderBindVisitor (from the last task).
You can resolve this with a static method that asks for a context or whatever,,,